Dispatcher/Constants/APIConstants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constants.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+struct APIConstants {
+    static let latestHeadlinesNewscatcherURL = "https://api.newscatcherapi.com/v2/latest_headlines"
```

Comment:
Basically the baseUrl is the same (https://api.newscatcherapi.com/v2/) and only the endpoint changes. So better have a baseUrl variable and then have different endpoints variables that you stick onto it when needed while you construct the url string when you build the request